01_classify_stratified/nbconverted/plot_mutation_results.py

```diff
@@ -37,183 +37,189 @@
 
 # load raw data
 results_df = au.load_stratified_prediction_results(results_dir, 'gene')
+
+# here we want to use compressed data for methylation datasets (27k and 450k)
+# the results in 02_classify_compressed/compressed_vs_raw_results.ipynb show that
+# performance is equal or slightly better for PCA compressed methylation data,
+# and it's much easier/faster to fit models on
+results_df = results_df[results_df.training_data.isin(['expression', 'rppa'])]
+
 print(results_df.shape)
+print(results_df.seed.unique())
+print(results_df.training_data.unique())
 results_df.head()
 
 
 # In[4]:
 
 
-expression_df = (
-    results_df[results_df.training_data == 'expression']
-        .drop(columns=['training_data'])
-)
-expression_results_df = au.compare_results(expression_df,
-                                           identifier='identifier',
-                                           metric='aupr',
-                                           correction=True,
-                                           correction_method='fdr_bh',
-                                           correction_alpha=SIG_ALPHA,
-                                           verbose=True)
-expression_results_df.rename(columns={'identifier': 'gene'}, inplace=True)
-expression_results_df.sort_values(by='p_value').head(n=10)
+# load compressed data for me_27k and me_450k
+compressed_results_df = au.load_compressed_prediction_results(results_dir, 'gene')
+print(compressed_results_df.seed.unique())
```

Comment:
Is there a reason you print out the seed and dimensionality each time?

Reply:
This was mainly for debugging, but I think it's useful to keep for now since I'm combining results from uncompressed and compressed data in a bit of an ad hoc way at the moment (using compressed data for methylation datasets and uncompressed data for other data types). These checks make sure I'm loading the right data (i.e. two replicates for all data types, PCA n=5000, etc) to make a fair comparison between data types.

I'll add a note to this script explaining what I'm checking and why, and I'll create an issue to clean this up in the future. I'm planning to re-organize these scripts so that the order they run in makes more sense (the compression analysis will come first, which will justify using compressed methylation data for subsequent analyses) but that'll be a future PR.